publications-2019.bib

```diff
@@ -1057,4 +1057,273 @@ @PhdThesis{MB_PhD_Thesis
+    archivePrefix={arXiv},
+    primaryClass={math.NA}
+}
+
+
+@misc{mulita2019quasioptimal,
+    title={Quasi-optimal mesh sequence construction through Smoothed Adaptive Finite Element Method},
+    author={Ornela Mulita and Stefano Giani and Luca Heltai},
+    year={2019},
+    eprint={1905.06924},
+    archivePrefix={arXiv},
+    primaryClass={math.NA}
+}
+
+
+@misc{klein2019sequential,
+    title={Sequential subspace optimization for recovering stored energy functions in hyperelastic materials from time-dependent data},
+    author={Rebecca Klein and Thomas Schuster and Anne Wald},
+    year={2019},
+    eprint={1911.04855},
+    archivePrefix={arXiv},
+    primaryClass={math.NA}
+}
+
+
+@misc{liu2019balancing,
+    title={Balancing truncation and round-off errors in practical FEM: one-dimensional analysis},
+    author={Jie Liu and Matthias M{\"o}ller and Henk M. Schuttelaars},
+    year={2019},
+    eprint={1912.08004},
+    archivePrefix={arXiv},
+    primaryClass={math.NA}
+}
+
+
+@misc{anselmann2019higher,
+    title={Higher order Galerkin-collocation time discretization with Nitsche's method for the Navier-Stokes equations},
+    author={Mathias Anselmann and Markus Bause},
+    year={2019},
+    eprint={1912.07426},
+    archivePrefix={arXiv},
+    primaryClass={math.NA}
+}
+
+
+@misc{holmgren2019hydrodynamic,
+    title={A Hydrodynamic Model of Movement of a Contact Line Over a Curved Wall},
+    author={Hanna Holmgren and Gunilla Kreiss},
+    year={2019},
+    eprint={1905.08788},
+    archivePrefix={arXiv},
+    primaryClass={physics.flu-dyn}
+}
+
+
+@misc{heltai2019priori,
+    title={A priori error estimates of regularized elliptic problems},
+    author={Luca Heltai and Wenyu Lei},
+    year={2019},
+    eprint={1911.02293},
+    archivePrefix={arXiv},
+    primaryClass={math.NA}
+}
+
+
+@misc{murphy2019moving,
+    title={A moving grid finite element method applied to a mechanobiochemical model for 3D cell migration},
+    author={Laura Murphy and Anotida Madzvamuse},
+    year={2019},
+    eprint={1903.09535},
+    archivePrefix={arXiv},
+    primaryClass={q-bio.CB}
+}
+
+
+@misc{alzetta2019multiscale,
+    title={Multiscale modeling of fiber reinforced materials via non-matching immersed methods},
+    author={Giovanni Alzetta and Luca Heltai},
+    year={2019},
+    eprint={1906.03881},
+    archivePrefix={arXiv},
+    primaryClass={math.NA}
+}
+
+
+@misc{alex2019modeling,
+    title={Modeling and simulation of heat source trajectories through phase-change materials},
+    author={Alexander Gary Zimmerman},
+    year={2019},
+    eprint={1909.08882},
+    archivePrefix={arXiv},
+    primaryClass={cs.CE}
+}
+
+
+@misc{he2019simulating,
+    title={Simulating the scattering of low-frequency Gravitational Waves by compact objects using the finite element method},
+    author={Jian-hua He},
+    year={2019},
+    eprint={1912.00325},
+    archivePrefix={arXiv},
+    primaryClass={gr-qc}
+}
+
+
+@misc{mulita2019quasioptimal,
```

Comment:
the publications starting from here and up to line 1267 appear to be duplicates.

Reply:
You're right, thanks for pointing those out.